src/main/java/core/basesyntax/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class BalanceHandler implements OperationHandler {
+    private static final String BALANCE = "b";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
+                                                        List<FruitTransaction> listOfFruits) {
```

Comment:
The method name 'getFruitFinalQuantity' is not very descriptive of what the method is supposed to do. Consider renaming the method to better reflect its purpose, such as 'applyBalanceOperation' or 'updateFruitQuantitiesBasedOnBalance'.